Add opus to MP4 direct play profile

solidsnake1298 opened the pull request:
**Changes**
Adds OPUS to the MP4 container direct play profile.  OPUS is supported for direct play in other containers, but not MP4.  The MP4 spec supports OPUS.

**Issues**
Resolves this user's issue with unnecessary transcoding.

https://forum.jellyfin.org/t-solved-direct-play-problem-with-mp4-and-certain-codecs-on-android-native-player

Maxr1998 commented:
Thanks for the PR. I think it would actually be preferable to merge this into master and cherry-pick the commit to the release branch later on. Would you mind fixing your earlier PR? Then, this one could be closed.

solidsnake1298 commented:
> Thanks for the PR. I think it would actually be preferable to merge this into master and cherry-pick the commit to the release branch later on. Would you mind fixing your earlier PR? Then, this one could be closed.

I did a noob move and deleted my fork so re-opening my previous PR is not possible anymore.  But I just changed the target branch on this one.

Maxr1998 commented:
I guess that works. Sorry for taking so long to reply.
